lib/ansible/modules/notification/syslogger.py

```diff
@@ -143,7 +143,7 @@ def run_module():
 
     # basically, do nothing.
     if module.check_mode:
-        return result
+        module.exit_json(**result)
```

Comment:
the result has nothing of value at this point, i would just remove 'supports check mode' and have this module be skipped